cloud/azure/bin/lib/bastion.sh

```diff
@@ -115,3 +115,69 @@ function bastion::get_connect_to_postgres_command() {
     yes | sudo apt-get install postgresql-client > /dev/null; \
     PGPASSWORD='${DB_PASSWORD}' psql -h ${1} -d postgres -U psqladmin@${1}"
 }
+
+#######################################
+# Get the command to restore db data.
+# More details on the psql version # https://www.postgresql.org/download/linux/ubuntu/
+# the pg_restore is symlinked to that via a pg_wrapper
+# Arguments:
+#   1: the postgres host to connect to 
+#   2: the vaultname where secrets are stored
+#   3: path of the dump data
+#######################################
+function bastion::get_pg_restore_command() {
+  local DB_PASSWORD=$(bastion::get_pg_password "${2}")
+  echo "export DEBIAN_FRONTEND='noninteractive'; \
+    yes | sudo apt-get update > /dev/null; \
+    yes | sudo apt-get install postgresql-14 > /dev/null; \
```

Comment:
this postgresql-14 stuff might need a lil attention. this should work but ideally we'd not pin it to a version i just got a lil losed in the symlink stuff so settled on this for the time being